tests/installation/bootloader_svirt.pm

```diff
@@ -8,14 +8,20 @@
 # without any warranty.
 
 use base "installbasetest";
-
 use strict;
 use warnings;
-
-use File::Basename;
-
 use testapi;
 use utils;
+use File::Basename;
+
+sub is_netinstall() {
+    if (!is_jeos && !get_var('ISO')) {
+        return 1;
+    }
+    else {
+        return 0;
+    }
+}
 
```

Comment:
Thanks will talk to him. It's part of the zkvm bootloader code on which I based svirt bootloader, thought it should work on KVM and Xen HVM.